lcviz/plugins/flatten/flatten.py

```diff
@@ -43,6 +46,7 @@ class Flatten(PluginTemplateMixin, DatasetSelectMixin, AddResultsMixin):
     uses_active_status = Bool(True).tag(sync=True)
 
     show_live_preview = Bool(True).tag(sync=True)
+    show_trend_preview = Bool(True).tag(sync=True)
```

Comment:
```suggestion
    show_trend_preview = Bool(False).tag(sync=True)
```
Take or leave this suggestion, but I think the default should be less cluttered and redundant. Especially since the trend color and live preview color are identical, there's a lot to see in the viewer if this is enabled by default.

Reply:
usually I actually prefer to see the trend over the preview 🤷‍♂️   Let's leave for now and revisit as we use it more.